{{cookiecutter.app_name}}_role/molecule/default/tests/test_app.yml

```diff
@@ -1,36 +1,45 @@
 ---
```

Comment:
Probably we should add somewhere a note like "Remember to adapt goss tests to your convenience" os something lilke that.
Cookiecutter Readme migth be the best place, but a TODO comment on the goss test should also work fine.